spec/unit/controller_spec.rb

```diff
@@ -284,5 +303,25 @@ class ControllerApplication < Flame::Application
 			inherited_controller = Class.new(ControllerController.with_actions)
 			inherited_controller.actions.should.equal ControllerController.actions
 		end
+
+		it 'should include actions from mixin module' do
+			controller_with_mixin = ControllerController.include(
+				ControllerController.with_actions(MixIn)
+			)
+			controller_with_mixin.actions.should.be include_actions(
+				MixIn.public_instance_methods
```

Comment:
In tests it is better to duplicate the information explicitly. If they then start to fall after the changes - there's nothing wrong with that, on the contrary, it's clearer what has changed, and whether it is necessary to change something.